src/main/java/edu/ucla/library/prl/harvester/handlers/ListJobsHandler.java

```diff
@@ -0,0 +1,48 @@
+
+package edu.ucla.library.prl.harvester.handlers;
+
+import org.apache.http.HttpStatus;
+
+import edu.ucla.library.prl.harvester.Job;
+import edu.ucla.library.prl.harvester.MediaType;
+import edu.ucla.library.prl.harvester.services.HarvestScheduleStoreService;
+
+import io.vertx.core.Handler;
+import io.vertx.core.Vertx;
+import io.vertx.core.http.HttpHeaders;
+import io.vertx.core.http.HttpServerResponse;
+import io.vertx.core.json.JsonArray;
+import io.vertx.ext.web.RoutingContext;
+
+/**
+ * A handler for listing jobs.
+ */
+public final class ListJobsHandler implements Handler<RoutingContext> {
+
+    /**
+     * A proxy to the harvest schedule store service.
+     */
+    private final HarvestScheduleStoreService myHarvestScheduleStoreService;
+
+    /**
```

Comment:
What additional information do you think is needed here?

Reply:
Just saying what this does for the generated javadocs (instead of having a blank value in them)... it doesn't have to be complicated: "Creates a new handler for job list requests." or something is fine.

Reply:
It just begins to seem redundant, since creating new instances is what constructors do. But okay.

Reply:
Yes, a lot of times it's evident what something does (at least the author), but I look at it as producing user-readable documentation (Javadocs).

Reply:
Since this is not a library, I think it makes more sense to evaluate the code comments on the basis of how well they assist developers on the project, rather than trying to "fill out" the HTML-ized javadoc. And actually, I don't understand why that is part of the criteria at all.

Reply:
I think javadocs assist the users (in this case developers) of the project, both new and old. You're lead here though, you're welcome to disregard my suggestion. :-)